app/helpers/kaui/date_helper.rb

```diff
@@ -22,5 +22,25 @@ def format_date(date, timezone)
     def truncate_millis(date_s)
       DateTime.parse(date_s).strftime('%FT%T')
     end
+
+    def current_time(time_zone=nil)
+      time_zone = @account.time_zone unless @account.nil? || time_zone.nil?
```

Comment:
I think it would be cleaner to pass the timezone instead of expecting that the instance variable is set (see also discussion here: https://stackoverflow.com/questions/6474596/should-rails-helpers-assume-an-instance-variable-exists-or-should-they-receive-t).